nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -264,6 +264,8 @@ subsonic-compatible api. Available as [navidrome](#opt-services.navidrome.enable
 
 - The `isabelle` package has been upgraded from 2020 to 2021
 
+- The `cozy-drive` package has been added. v3.29.0.
+
```

Comment:
```suggestion
```

I don't think this change is significant to over 99% of the users and so does not require a changelog entry. Also we usually only announce new package if they are really important to a big group of users.